src/users/__tests__/userSelectors-test.js

```diff
@@ -208,3 +209,28 @@ describe('getUsersById', () => {
     expect(result).toEqual(expectedResult);
   });
 });
+
+describe('getOtherUsers', () => {
+  test('returns all users but current user', () => {
+    const state = deepFreeze({
+      users: [
+        { id: 1, email: 'abc@example.com', isActive: true },
```

Comment:
Do include only values that are needed for the test.
Do you think `isActive` is used anywhere?

Reply:
No, I'l just remove it.